### PR TITLE
Revert intel-media-driver

### PR DESCRIPTION
**Pull request by Gottox**
- Revert "intel-media-driver: update to 22.1.1"
- Revert "intel-gmmlib: update to 22.0.1"

This works around a problem with firefox which crashes with the following message:

```
libva info: VA-API version 1.13.0
libva info: Trying to open /usr/lib64/dri/iHD_drv_video.so
libva info: Found init function __vaDriverInit_1_13
   readlink -errno 13
```

#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Comment by daerich** [comment]
Just because I'm interested: Do you have a bugtracking link, since Iam running Firefox (non esr) with VA and had none of the described issues so far.

**Comment by Vistaus** [comment]
> Just because I'm interested: Do you have a bugtracking link, since Iam running Firefox (non esr) with VA and had none of the described issues so far.

I can't reproduce this issue either.

**Comment by daerich** [comment]
> Just because I'm interested: Do you have a bugtracking link, since Iam running Firefox (non esr) with VA and had none of the described issues so far.

Just to report back, after testing further and running a couple of power intensive tasks(compiling llvm13 and Linux5.16.8) and running Firefox in Picture in Picture I noticed serious performance degradations and a rise in compile times(+1hrs for Linux with no changed config), I don't know if it's because of wear, but seeing Arch still using the old version and some reporting crashes in this thread, I am more inclined to attribut this to VAAPI

File: common/shlibs [hunk 1]
```diff
@@ -3874,8 +3874,9 @@ libnvpair.so.3 zfs-2.0.3_2
 libgmio.so gmio-0.4.1_1
 libjsonnet.so.0 jsonnet-0.14.0_2
 libjsonnet++.so.0 jsonnet-0.14.0_2
-libigdgmm.so.12 intel-gmmlib-22.0.1_1
+libigdgmm.so.11 intel-gmmlib-21.3.1_1
 libigfxcmrt.so.7 intel-media-driver-21.3.5_1
+libigdgmm.so.11 intel-gmmlib-19.4.1_1
```
Comment (daerich) [inline common/shlibs:3879]: This reintroduces a duplicate line in "common/shlibs"; I don't know if this is desired but I'd like to point this out

**Review by daerich — COMMENTED** [review]
This reintroduces a duplicate line

**Comment by cinerea0** [comment]
This was [reported on reddit](https://old.reddit.com/r/voidlinux/comments/sodbxo/firefox_stopped_playing_videos_with_hw/) as well and appears to be due to Firefox's hardware rendering.

**Comment by Vistaus** [comment]
Looks like this was a Firefox issue that's been solved in 97: https://www.ghacks.net/2022/02/11/how-to-fix-hardware-video-acceleration-in-firefox-on-linux/
97 is already available in Void, so I think this PR can be closed.

**Comment by Gottox** [comment]
Closed, fixed by updating ff to 97

**Comment by steinex** [comment]
I still have the exact same issue with Firefox 97 running on `MOZ_ENABLE_WAYLAND=1` and `media.ffmpeg.vaapi.enabled` set to true. Any ideas?

**Comment by Gottox** [comment]
Let's reopen the PR and get this merged. I have no

**Comment by cinerea0** [comment]
I'm experiencing the same problem, I was only able to solve it by setting `media.ffmpeg.vaapi.enabled` to false. I assumed that the option was no longer supported, but if we can keep it working that would be ideal.

**Pull request merged**